src/components/Header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import '../../style/Header/header.css'
+import SearchForm from "./SearchForm";
+const Header = () => (
+
+    <div className={'header'}>
+        <div className={'content'}>
+            <div className={'first_line'}>
+                <label className={'logo'}><span>netflix</span>roulette</label>
+                <a className={'add_movie'}>+ ADD MOVIE</a>
```

Comment:
this doesn't seem like an `anchor` I think it's a button that when clicked opens the modal